ci/test/mkpipeline.sh

```diff
@@ -45,4 +45,6 @@ steps:
   - label: mkpipeline
     command: bin/ci-builder run stable bin/pyactivate --dev -m ci.test.mkpipeline
     priority: 2
+    agents:
```

Comment:
The problem is that the general Linux queue doesn’t trigger autoscaling :/

Reply:
oh -- I thought you did this on purpose and I was saying it was probably fine! but maybe you do want to change it to `linux-x86_64`?

Reply:
Oh, right! I convinced myself that this was fine because we always run at least one builder instance, so this will always get processed _eventually_, and when there are a bunch of live agents should be a bit lower latency to target the general `linux` queue.